internal/build/docker.go

```diff
@@ -195,10 +195,13 @@ func (l *localDockerBuilder) execStepsOnImage(ctx context.Context, baseDigest di
 	imageWithSteps := string(baseDigest)
 	for _, s := range steps {
 		cId, err := l.startContainer(ctx, imageWithSteps, &s)
+		if err != nil {
```

Comment:
I'm not sure if the ignored errs here and below were intentional or if I'm breaking things by checking them...I'm happy to change these back to ignored and add a comment for posterity

Reply:
aah no you are totally correct, those should be handled, i messed up 😳